lib/redis_client/cluster/command.rb

```diff
@@ -24,7 +24,7 @@ class << self
         def load(nodes)
           cmd = errors = nil
 
-          nodes&.each do |node|
+          nodes&.shuffle_each do |node|
```

Comment:
> It seems that binding variables in the closure didn't work.

@supercaracal were you referring to this closure? I'm trying to replicate it locally but `bundle exec rake test` completes without the same failures.

Reply:
Sorry, I misunderstood. I'll look into the CI failure later.

Reply:
>Sorry, I misunderstood. I'll look into the CI failure later.

No worries! I'll look into it after work too 🙇 thanks for the extra set of eyes.

Reply:
The errors occur in my local WSL2 environment too. All test cases depend on Docker containers in ubuntu machines of GitHub actions.

Since `Command` becomes to depend `Node`, it needs more complex preparation like this:

```ruby
      def setup
        config = ::RedisClient::ClusterConfig.new(
          nodes: TEST_NODE_URIS,
          fixed_hostname: TEST_FIXED_HOSTNAME,
          **TEST_GENERIC_OPTIONS
        )
        node_info_list = ::RedisClient::Cluster::Node.load_info(config.per_node_key)
        if TEST_FIXED_HOSTNAME
          node_info_list.each do |info|
            _, port = ::RedisClient::Cluster::NodeKey.split(info.node_key)
            info.node_key = ::RedisClient::Cluster::NodeKey.build_from_host_port(TEST_FIXED_HOSTNAME, port)
          end
        end
        node_addrs = node_info_list.map { |info| ::RedisClient::Cluster::NodeKey.hashify(info.node_key) }
        config.update_node(node_addrs)
        @raw_clients = ::RedisClient::Cluster::Node.new(config.per_node_key, node_info_list: node_info_list)
        @empty_clients = ::RedisClient::Cluster::Node.new([], node_info_list: node_info_list)
      end
```

Reply:
I think it would be better to shuffle at the following step:
https://github.com/redis-rb/redis-cluster-client/blob/c22ca367873435fb93f21f4aaaa418daed1760c3/lib/redis_client/cluster/router.rb#L26

I'd say that we can keep to less dependencies between class and class.

Reply:
That is a fair point, the `setup` looks like it will get fairly complicated. I'll make the suggested adjustments 🙇

Reply:
@supercaracal Done in https://github.com/redis-rb/redis-cluster-client/pull/239/commits/5e3d396ea56a0cc42c5d757ab9f21eee30d25ebd.  🏓 

I deliberated between creating a new `shuffled_nodes` instance method or just using `@nodes.to_a.shuffle` since it is fairly one-off usage. I'll defer to your maintainer review on which is preferred in this codebase.